micro-ui/web/micro-ui-internals/packages/modules/workbench/src/components/MultiSelect.js

```
@@ -139,7 +169,7 @@ const CustomSelectWidget = (props) => {
   };
   const handleViewMoreClick = (detail) => {
     const schemaCode = detail?.schemaCode;
-    const [moduleName, masterName] = schemaCode.split(".");
+    const [moduleName, masterName] = schemaCode?.split(".");
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)